src/python/pants/backend/native/register.py

```diff
@@ -25,6 +27,7 @@ def build_file_aliases():
     targets={
       CLibrary.alias(): CLibrary,
       CppLibrary.alias(): CppLibrary,
+      ExternalNativeLibrary.alias(): ExternalNativeLibrary
```

Comment:
We should have consistent formatting of commas, at least in the native backend. I'll use this style in the future so we don't have any conflict, but also, it's generally a very good idea to maintain the same style as the rest of the file/directory, no matter where it is. Style changes aren't usually big enough to require a change on their own, and many people may not point it out for fear of being nitpicky, and it's really helpful if you can make sure to uphold whatever style is used in the current file so people in the future editing this file are clear on which to use.